politeiad/backendv2/tstorebe/store/store.go

```diff
@@ -98,6 +126,10 @@ type BlobKV interface {
 	// was returned for all provided keys.
 	Get(keys []string) (map[string][]byte, error)
 
+	// Tx returns a new database transaction as well as the cancel
+	// function that releases all resources associated with it.
+	Tx() (Tx, func(), error)
```

Comment:
maybe clarify under what conditions this `func` needs to be called,

why can't we call it from `Commit` / `Rollback` methods automatically ?